curated_transformers/tokenization/xlmr_tokenizer.py

```diff
@@ -73,87 +73,59 @@ def _fairseq_to_sentencepiece(piece_id: int):
             return piece_id - _XLMR_FAIRSEQ_OFFSET
 
 
-class XlmrTokenizer(SentencePieceTokenizer, FromPretrainedHFTokenizer):
+class XlmrTokenizer(SentencePieceTokenizer, LegacyFromHFHub):
```

Comment:
Do we want to change the camel-case here (and for the other legacy tokenizers) to make it consistent with the other tokenizer classes?

Reply:
Yeah, we should do that, I'll do this in a follow-up PR.